jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListSavedFiltersPresenter.java

```diff
@@ -0,0 +1,131 @@
+import static org.jbpm.workbench.pr.model.ProcessInstanceDataSetConstants.*;
+
+@Dependent
+@WorkbenchScreen(identifier = PROCESS_INSTANCE_LIST_SAVED_FILTERS_SCREEN)
+public class ProcessInstanceListSavedFiltersPresenter extends SavedFiltersPresenter {
+
+    protected static final String TAB_ACTIVE = PROCESS_INSTANCES_WITH_VARIABLES_INCLUDED_LIST_PREFIX + "_0";
+    protected static final String TAB_COMPLETED = PROCESS_INSTANCES_WITH_VARIABLES_INCLUDED_LIST_PREFIX + "_1";
+    protected static final String TAB_ABORTED = PROCESS_INSTANCES_WITH_VARIABLES_INCLUDED_LIST_PREFIX + "_2";
+
+    private Constants constants = Constants.INSTANCE;
+
+    @WorkbenchMenu
+    public Menus getMenus() { //It's necessary to annotate with @WorkbenchMenu in subclass
+        return super.getMenus();
+    }
+
+    @Override
+    public String getGridGlobalPreferencesKey() {
+        return PROCESS_INSTANCES_WITH_VARIABLES_INCLUDED_LIST_PREFIX;
+    }
+
+    /*-------------------------------------------------*/
+    /*---              DashBuilder                   --*/
+    /*-------------------------------------------------*/
+    @Override
+    public FilterSettings createTableSettingsPrototype() {
+        FilterSettingsBuilderHelper builder = FilterSettingsBuilderHelper.init();
+        builder.initBuilder();
+
+        builder.dataset(PROCESS_INSTANCE_DATASET);
+        builder.filterOn(true,
+                         true,
+                         true);
+        builder.tableOrderEnabled(true);
+        builder.tableOrderDefault(COLUMN_START,
+                                  SortOrder.DESCENDING);
+        builder.tableWidth(1000);
+
+        final FilterSettings filterSettings = builder.buildSettings();
+        filterSettings.setUUID(PROCESS_INSTANCE_DATASET);
+        return filterSettings;
+    }
+
+    private FilterSettings createStatusSettings(final Integer state) {
+        FilterSettingsBuilderHelper builder = FilterSettingsBuilderHelper.init();
+        builder.initBuilder();
+
+        builder.dataset(PROCESS_INSTANCE_DATASET);
+
+        builder.filter(equalsTo(COLUMN_STATUS,
+                                state));
+
+        builder.filterOn(true,
+                         true,
+                         true);
+        builder.tableOrderEnabled(true);
+        builder.tableOrderDefault(COLUMN_START,
+                                  SortOrder.DESCENDING);
+
+        return builder.buildSettings();
+    }
+
+    public FilterSettings createActiveTabSettings() {
+        return createStatusSettings(ProcessInstance.STATE_ACTIVE);
+    }
+
+    public FilterSettings createCompletedTabSettings() {
+        return createStatusSettings(ProcessInstance.STATE_COMPLETED);
+    }
+
+    public FilterSettings createAbortedTabSettings() {
+        return createStatusSettings(ProcessInstance.STATE_ABORTED);
+    }
+
+    @Override
+    public void initDefaultFilters() {
+        //Filter status Active
```

Comment:
These comments are present in some presenters and in some not

Reply:
Added comments to all places.